.github/workflow_scripts/run_benchmark.sh

```diff
@@ -14,31 +14,31 @@ source $(dirname "$0")/env_setup.sh
 setup_benchmark_env
 
 /bin/bash CI/bench/generate_bench_config.sh $MODULE $PRESET $BENCHMARK $TIME_LIMIT $BRANCH_OR_PR_NUMBER
-agbench run $MODULE"_cloud_configs.yaml" --wait
+agbench run $MODULE"_cloud_configs.yaml" --dev-branch https://github.com/autogluon/autogluon-bench.git#master --wait
 
 # If it is a PR, fetch the cleaned file of master-evaluation
 if [ $BRANCH_OR_PR_NUMBER != "master" ]
 then
     # Capture the name of the file, rename it and store it in ./results
-    master_cleaned_file=$(aws s3 ls s3://autogluon-ci-benchmark/cleaned/master/latest/ | awk '{print $NF}')
+    master_cleaned_file=$(aws s3 ls s3://autogluon-ci-benchmark/cleaned/$MODULE/master/latest/ | awk '{print $NF}')
     new_master_cleaned_file="master_${master_cleaned_file}"
-    aws s3 cp --recursive s3://autogluon-ci-benchmark/cleaned/master/latest/ ./results
+    aws s3 cp --recursive s3://autogluon-ci-benchmark/cleaned/$MODULE/master/latest/ ./results
```

Comment:
Is it correct to assume that the `MODULE` here pertains to the categories of `tabular`, `multimodal`, and `timeseries`?

Reply:
Yes, the assumption is correct.